x-pack/plugins/lists/server/services/exception_lists/create_endoint_event_filters_list.ts

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I think there is a typo on this file name.
create_endoint_event_filters_list.ts -> create_endpoint_event_filters_list.ts

Reply:
Thanks @dasansol92 . I'll fix this in the next commit if we end up keeping it (will also be adding tests for it). I say "if we keep it" because we might not actually use it if we decide to create the list via an explicit API call rather than to have it created automagically